src/routes/visits.js

```diff
@@ -21,4 +21,19 @@ const validatorVisitID = Joi.object({
 });
 router.get('/:id', validator.params(validatorVisitID), wrapper(getVisitByID));
 
+const validatorUpdateVisit = Joi.object({
+	clientId: Joi.objectId().required(),
```

Comment:
I don't think it's a good idea to use only one, :id and clientId have same value, but :id is used for finding visit in database and clientId to be part of the body in PUT request

Reply:
Okay, currently, I'm seeing it. However, from that, I've got one question more.
Why do u change the ID of the document in the database? 
It's a property without changes.

Reply:
Okay, currently, I'm seeing it. However, from that, I've got one question more.
Why do u change the ID of the document in the database?
It's a property without changes.

Reply:
Okay, why do u try to change ID? It's the primary key.
In this way, u can break the principle "Data integrity".

Reply:
I do not try to change ID, if you mean I put the `clientId` field in body - I did it because if I don't, PUT request would just delete the field clientId from document in database.
Or I don't really understand what you ask me to do

Reply:
Oh, yes. My bad. I thought you try to change the primary key (id) of the visit, but not. It's a foreign key (to clients).